model/__init__.py

```diff
@@ -573,6 +575,9 @@ def deploy_delaying_proxy(self):
     def kubernetes_cluster_config(self):
         return self.snd.kubernetes_cluster_config
 
+    def disable_webhook_for_pr(self):
+        return self.snd.disable_webhook_for_pr
```

Comment:
while you may use SND's behaviour to return None for absent values, it might still be a good idea to use the more explicit `self.raw.get('disable_webhhok_for_pr', False)` - in particular, this would avoid introducing yet another dependency towards SND; if I properly recall, we all agreed on removing SND eventually ;-)